packages/react-native-reanimated/android/src/main/cpp/reanimated/CMakeLists.txt

```diff
@@ -1,14 +1,14 @@
 cmake_minimum_required(VERSION 3.8)
 
 file(GLOB_RECURSE REANIMATED_COMMON_SOURCES CONFIGURE_DEPENDS "${REANIMATED_COMMON_DIR}/*.cpp")
-file(GLOB_RECURSE SOURCES_ANDROID CONFIGURE_DEPENDS "${ANDROID_SRC_DIR}/main/cpp/*.cpp")
+file(GLOB_RECURSE SOURCES_ANDROID CONFIGURE_DEPENDS "${REANIMATED_ANDROID_DIR}/*.cpp")
```

Comment:
this needs to be cleaned up since `REANIMATED_ANDROID_DIR` might be confusing (at first glance it looks like it points to `packages/react-native-reanimated/android`)